application/src/modules/homepage/HomepageState.js

```diff
@@ -55,6 +66,7 @@ export default function HomepageStateReducer(state = initialState, action = {})
     case NOTIFICATION_RESPONSE:
       // We got a notification update so let's update the state and then restart the timer.
       return loop(
+        // TODO(@iprunache) stop triggering a render for every poll when the payload does not change; happens with immutable too.
```

Comment:
What i remember from the initial research on React + Redux is that a render occurs when the state changes. And the reducer uses some algorithm to check if the new state is `equal` to the previous one. I've seen recommendations to use immutable objects since equals is fast for them but I may be using them wrong - creating a new immutable object every time.

What led me to realise that renders occur with every fetch is that the navigation sound is played every time :) And that's because the navigation sound is played from the renderer.